packages/react-sdk/docusaurus/docs/React/01-basics/07-ui-customization-options.mdx

```diff
@@ -14,22 +14,22 @@ Here is a list of things still to address on this page:
 
 ## Introduction
 
-There are different ways to use the React Video SDK to build out UI. It comes with powerful built-in video components ready for use.
+There are different ways to use the React Video SDK to build out UI. It comes with powerful built-in components ready for use.
 In addition, it allows for styling and custom theming options with custom CSS.
 
-For custom use cases it is also possible to only use the low-level client to provide completely custom UI to fully tailor the SDK towards your needs.
+For custom use cases it is also possible to create your own components to fully tailor the SDK towards your needs. Please note that we suggest not to override a few, core components as those contain complex, low-level logic related to performance, the list of these components can be found in the [UI Components overview](../03-ui/01-overview.mdx).
 
 This page guides you through all of them with increasing levels of customization.
 
-### Using pre-built components
+## Using pre-built components
 
 The SDK comes with a lot of pre-built components for the different elements of the UI.
 
 :::tip
 For a list of all available UI components head over to the [Components](../../ui/01-ui-overview) section.
 :::
 
-### Theming with CSS
+## Theming with CSS
 
 If you only want to customize the styling of the available components, there is a wide range of CSS variables that you can override.
 With that, you can achieve a significant amount of theming in your video product.
```

Comment:
The following section has a grammatic mistake:

Here are a few examples: -> Here are few examples:

Reply:
I think this page, in general, will need some more work, as mentioned by @DaemonLoki. Feel free to add those changes @MartinCupela, or we can revisit them outside of this PR.

Reply:
Agree with Zita. 

It surely requires info (e.g. I didn't know about the `str-video` thing up until yesterday. So there's surely more things needed, I just wanted to give a basic skeleton to have a start.

Please always feel free to fill this with relevant wisdom! :)

Reply:
> The following section has a grammatic mistake:
> 
> Here are a few examples: -> Here are few examples:

I'd argue that the first one is actually correct?

Reply:
few vs. a few (https://www.grammarly.com/blog/few-a-few/) - not sure which you had in mind @DaemonLoki